go.mod

```diff
@@ -1,17 +1,10 @@
-module github.com/openlyinc/openlynt
+module go.openly.dev/openlynt
 
 go 1.13
 
 require (
-	github.com/Masterminds/goutils v1.1.0 // indirect
-	github.com/Masterminds/semver v1.5.0 // indirect
 	github.com/Masterminds/sprig v2.22.0+incompatible
 	github.com/golangci/revgrep v0.0.0-20180812185044-276a5c0a1039
-	github.com/google/uuid v1.1.1 // indirect
-	github.com/huandu/xstrings v1.3.0 // indirect
-	github.com/imdario/mergo v0.3.8 // indirect
-	github.com/mitchellh/copystructure v1.0.0 // indirect
-	github.com/stretchr/testify v1.4.0 // indirect
-	golang.org/x/crypto v0.0.0-20200109152110-61a87790db17 // indirect
+	github.com/openlyinc/openlynt v0.0.1
```

Comment:
should this be the vanity url?

Reply:
That's the problem, until I publish this I can't update this dependency. Unless I'm missing something...